fastlane/spec/actions_specs/increment_version_number_action_spec.rb

```diff
@@ -89,16 +130,18 @@
         end.to raise_error("Please pass the path to the project, not the workspace")
       end
 
-      it "raises an exception when unable to calculate new version" do
-        expect(Fastlane::Actions).to receive(:sh)
-          .with(/agvtool what-marketing-version/, any_args)
-          .once
-          .and_return("00000")
-        expect do
-          Fastlane::FastFile.new.parse("lane :test do
-            increment_version_number
-          end").runner.execute(:test)
-        end.to raise_error("Your current version (00000) does not respect the format A.B.C")
+      ["A", "1.2.3.4"].each do |version|
```

Comment:
Would this also apply for e.g. `1.2.3-pre` when increasing the patch version?

Reply:
Good question. I don't handle this case so indeed this would raise an error.
I'll include it in the scenarios as well.

Reply:
Ok, so the version number really has to be 2 or 3 digits separated by dots, correct?

Reply:
correct

Reply:
I added another test case for `1.2.3-pre`